etc/kayobe/kolla-image-tags.yml

```
@@ -4,25 +4,25 @@
 # where the key is the OS distro and the value is the tag to deploy.
 kolla_image_tags:
   openstack:
-    rocky-9: 2025.1-rocky-9-20250730T105631
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20250730T105631
   iscsid:
-    rocky-9: 2025.1-rocky-9-20251013T135156
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20251013T135156
   keystone:
-    rocky-9: 2025.1-rocky-9-20251031T094539
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20251031T094539
   kolla_toolbox:
-    rocky-9: 2025.1-rocky-9-20250905T151507
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20250905T151507
   magnum:
-    rocky-9: 2025.1-rocky-9-20251016T073648
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20251016T073648
   ovn:
-    rocky-9: 2025.1-rocky-9-20250915T100822
+    rocky-9: 2025.1-rocky-9-20251208T092405
   prometheus_libvirt_exporter:
-    rocky-9: 2025.1-rocky-9-20250822T151934
+    rocky-9: 2025.1-rocky-9-20251208T092405
     ubuntu-noble: 2025.1-ubuntu-noble-20250822T151934
   rabbitmq:
-    rocky-9: 2025.1-rocky-9-20250905T151507
+    rocky-9: 2025.1-rocky-9-20251208T092405
```

the `openstack:` tag is the default tag for each OS. If the tags are identical, that is the only one that needs to be set, and all the others can be removed:


```suggestion
    ubuntu-noble: 2025.1-ubuntu-noble-20250730T105631
  iscsid:
    ubuntu-noble: 2025.1-ubuntu-noble-20251013T135156
  keystone:
    ubuntu-noble: 2025.1-ubuntu-noble-20251031T094539
  kolla_toolbox:
    ubuntu-noble: 2025.1-ubuntu-noble-20250905T151507
  magnum:
    ubuntu-noble: 2025.1-ubuntu-noble-20251016T073648
  prometheus_libvirt_exporter:
    ubuntu-noble: 2025.1-ubuntu-noble-20250822T151934
  rabbitmq:
```

(I'm not sure if I got the spacing right in the suggestion above, the GitHub UI is not my text editor of choice)

---

thanks for the tip Alex. I will do container images in a separate PR